recipes-extended/linux-input/linux-input_1.6.0.bb

```diff
@@ -1,4 +1,4 @@
-SRC_URI = "https://svwh.dl.sourceforge.net/project/linuxconsole/linuxconsoletools-${PV}.tar.bz2 \
+SRC_URI = "https://newcontinuum.dl.sourceforge.net/project/linuxconsole/linuxconsoletools-${PV}.tar.bz2 \
```

Comment:
I think you should use "https://download.sourceforge.net/project/linuxconsole/linuxconsoletools-${PV}.tar.bz2" so you go through the load balancer/redirector instead of hardcoding a particular server.